applications/nrf_desktop/src/modules/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-target_sources_ifdef(CONFIG_CAF_BLE_BOND_SUPPORTED
+target_sources_ifdef(CONFIG_CAF_BLE_BOND_APP_SPECIFIC
```

Comment:
Why nrf_desktop source file was included using CAF option? That seems strange - option should be specific to the application.

Reply:
Introduced in a separate PR: https://github.com/nrfconnect/sdk-nrf/pull/6562